flowmachine/flowmachine/core/server/query_schemas/flows.py

```diff
@@ -50,6 +50,18 @@ def _flowmachine_query_obj(self):
         return RedactedFlows(flows=Flows(loc1, loc2, join_type=self.join_type))
 
 
+class InflowsExposed(FlowsExposed):
```

Comment:
Ok, next step is to make this be the redacted inflow of the unredacted flows obj.

Reply:
Done now